sdk/search/azure-search-documents/azure/search/documents/__init__.py

```diff
@@ -24,162 +24,189 @@
 #
 # --------------------------------------------------------------------------
 
-from ._version import VERSION
-
-__version__ = VERSION
-
 from ._index import (
     AutocompleteQuery,
     IndexAction,
     IndexDocumentsBatch,
     IndexingResult,
     SearchIndexClient,
-    SearchQuery,
     SearchItemPaged,
+    SearchQuery,
     SuggestQuery,
     odata,
 )
-
 from ._service import SearchServiceClient
-from ._service._models import PatternAnalyzer, PatternTokenizer
 from ._service._generated.models import (
-    Field,
-    TextWeights,
-    ScoringFunction,
-    DistanceScoringFunction,
-    FreshnessScoringFunction,
-    MagnitudeScoringFunction,
-    TagScoringFunction,
-    DistanceScoringParameters,
-    FreshnessScoringParameters,
-    MagnitudeScoringParameters,
-    TagScoringParameters,
-    ScoringProfile,
-    CorsOptions,
-    Suggester,
-    RegexFlags,
     Analyzer,
-    CustomAnalyzer,
-    StandardAnalyzer,
-    StopAnalyzer,
-    Tokenizer,
-    ClassicTokenizer,
-    EdgeNGramTokenizer,
-    KeywordTokenizer,
-    MicrosoftLanguageStemmingTokenizer,
-    MicrosoftLanguageTokenizer,
-    NGramTokenizer,
-    StandardTokenizer,
-    UaxUrlEmailTokenizer,
-    TokenFilter,
+    AnalyzeRequest,
+    AnalyzeResult,
     AsciiFoldingTokenFilter,
+    AzureActiveDirectoryApplicationCredentials,
+    CharFilter,
```

Comment:
I think in last review meeting, we decide to organize the models into different groups, e.g. Tokenizers, Analyzers, Filters, right?

Reply:
Yes but that has not been settled and there other changes under discussion as well, everyone is holding off making those changes just yet.